Quantum-Control-Applications/ESR/T1.py

```diff
@@ -0,0 +1,172 @@
+from qm import SimulationConfig
```

Comment:
I would add a docstring at the beginning of each file explaining what it does. I know it is on the readme but when you downloaded the files and start working with it, it is more convenient to have it here as well :)